tests/PageObjects/RemoteMic/RemoteMicMainPage.ts

```diff
@@ -65,4 +69,53 @@ export class RemoteMicMainPagePO {
   public async pressReadyOnRemoteMic() {
     await this.readyButton.click();
   }
+
+  public get gameCodeInput() {
+    return this.page.getByTestId('game-code-input');
+  }
+
+  public async expectGameCodeToBe(gameCode: string) {
+    await expect(this.gameCodeInput).toHaveValue(gameCode);
+  }
+
+  public get connectButton() {
+    return this.page.getByTestId('connect-button');
+  }
+
+  public async expectPlayerToBeConnected() {
+    await expect(this.connectButton).toContainText('CONNECTED', { ignoreCase: true });
+  }
+
+  public async expectPlayerToBeDisconnected() {
+    await expect(this.connectButton).toContainText('DISCONNECTED', { ignoreCase: true });
+  }
+
+  public async expectConnectButtonActivityToBeAs(buttonActivity: 'enabled' | 'disabled') {
```

Comment:
being `disabled` or `enabled` is a state of the button, the method should rather be called `expectConnectButtonToBe`